config/initializers/log_tags.rb

```diff
@@ -0,0 +1,9 @@
+Rails.application.configure do
+
+  config.log_tags = [
+    lambda { |request| "origin=#{request.ip}" },
+    lambda { |_| "tid=#{Thread.current.object_id}" },
+    lambda { |request| "request_id=#{request.uuid}" }
+  ]
+
+end
```

Comment:
If you're renaming the file anyway you could add the missing trailing newline :)